sdk/src/main/java/com/hedera/hashgraph/sdk/FreezeTransaction.java

```diff
@@ -86,14 +86,30 @@ void validateChecksums(Client client) throws BadEntityIdException {
         return FreezeServiceGrpc.getFreezeMethod();
     }
 
+    void initFromTransactionBody() {
+        var body = sourceTransactionBody.getFreeze();
+        startHour = body.getStartHour();
+        startMinute = body.getStartMin();
+        endHour = body.getEndHour();
+        endMinute = body.getEndMin();
+    }
+
+    FreezeTransactionBody.Builder build() {
+        var builder = FreezeTransactionBody.newBuilder();
```

Comment:
This could be `return FreezeTransactionBody.newBuilder().setStartHour(startHour)...;`